### ISSUE TITLE
Unable to resolve module @clerk/shared/authorization 

### ISSUE THREAD

**Issue by yessineZ** [clerk/javascript#4177]
### Preliminary Checks

- [X] I have reviewed the documentation: https://clerk.com/docs

- [X] I have searched for existing issues: https://github.com/clerk/javascript/issues

- [X] I have not already reached out to Clerk support via email or Discord (if you have, no need to open an issue here)

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Clerk. Please ask questions in our Discord community: https://clerk.com/discord.

### Reproduction

https://github.com/yessineZ/MistyUberr

### Publishable key

pk_test_Y2xldmVyLWdyb3VwZXItNjIuY2xlcmsuYWNjb3VudHMuZGV2JA

### Description

Hello , i am working on reactNative project and i  faced a problem when i try to add clerk  to do the authentication feature , when i wrap the application with clerkProvider and clerkLoaded in _layout file under the app folder , i try to start the project i see this error : 
Unable to resolve module @clerk/shared/authorization from C:\Users\yessi\Desktop\MOBILE LEARNING\MistyUber\node_modules\@clerk\clerk-react\dist\internal.js: @clerk/shared/authorization could not be found within the project or in these directories:
  node_modules
  62 |
  63 | // src/hooks/useAuth.ts
> 64 | var import_authorization = require("@clerk/shared/authorization");
     |                                     ^
  65 | var import_react5 = require("react");
  66 |
  67 | // src/errors/messages.ts
and when i remove ClerkProvider and ClerkLoaded the error disappear and the application work fine , i followed the documentation of clerk with reactNative and i installed all the packages needed , thank you and i am waiting for your help .

### Environment

```Shell
EXPO_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_Y2xldmVyLWdyb3VwZXItNjIuY2xlcmsuYWNjb3VudHMuZGV2JA
```

**Comment by mewc** [comment]
+1 this is happening atm.




```
"@clerk/clerk-expo@^2.2.12":
  version "2.2.12"
  resolved "https://registry.yarnpkg.com/@clerk/clerk-expo/-/clerk-expo-2.2.12.tgz#5ac92f66a7b15d49e5e5d04f775fe6caa0941184"
  integrity sha512-muFZB4np4lachbwwZRj+9nR6DLRPh5AdVNndk73sszCrhwx5j5BjN3YC06r/DJYnhfN4RorFZzq6qeVs+XCHpA==
  dependencies:
    "@clerk/clerk-js" "5.22.1"
    "@clerk/clerk-react" "5.9.0"
    "@clerk/shared" "2.8.0"
    "@clerk/types" "4.21.0"
    base-64 "^1.0.0"
    react-native-url-polyfill "2.0.0"
    tslib "2.4.1"
    ```
    
    i've reverted back to 
    
    ```
    "@clerk/clerk-expo@^2.0.0":
  version "2.2.4"
  resolved "https://registry.npmjs.org/@clerk/clerk-expo/-/clerk-expo-2.2.4.tgz"
  integrity sha512-ey7uPU+xFr8lx70Lq5K22zchi/qZxgFWqn6oiK3VQ93PdIsJoTyiHePEFZDVBdquBgv30bAm2aX7pnSkGpggPg==
  dependencies:
    "@clerk/clerk-js" "5.17.0"
    "@clerk/clerk-react" "5.4.5"
    "@clerk/shared" "2.5.5"
    "@clerk/types" "4.16.0"
    base-64 "^1.0.0"
    react-native-url-polyfill "2.0.0"
    tslib "2.4.1"
    ```
    
    so i can continue on

**Comment by hyochan** [comment]
Downgrading to `2.0.0` works as I posted in #4178

**Comment by yessineZ** [comment]
thank you guys it worked when i downgrade to  2.2.10

**Comment by yessineZ** [comment]
thank you guys it worked when i downgrade to 2.2.10

**Comment by LekoArts** [comment]
This should be fixed with https://github.com/clerk/javascript/pull/4172 - once https://github.com/clerk/javascript/pull/4179 is merged a new version of `@clerk/expo` will be released.

**Comment by clerk-cookie** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
